compiler/modules/bir/bir.bal

```diff
@@ -83,6 +104,8 @@ public type FunctionCode record {|
     Register[] registers = [];
     # Single-entry single-exit regions
     Region[] regions = [];
+    # TODO:index in bir:Module functions 
+    [int, FunctionCode][] childAnnonFunctions = [];
```

Comment:
Spelling: `Annon`

Reply:
I think we can get rid of this.

Reply:
Fixed. Decided to get rid of the `childAnonFunctions` fields completely since nothing is using it